voting-dapp/src/components/MainVoter.js

```diff
@@ -0,0 +1,96 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types'
+
+import {
+  Typography,
+  FormControl,
+  InputLabel,
+  Select,
+  MenuItem,
+  Divider,
+} from '@mui/material';
+import AddIcon from '@mui/icons-material/Add';
+
+// props needed : proposals
+function MainVoter({ step, hasVoted }) {
+
+  return (
+    <>{/* TODO : use step label */}
+      <Typography component="h2" variant="h4" align="center">
+        Main Voter component
+      </Typography>
+      {/* Conditionnal display depending on the current status */}
+      {step == 0 &&
+        /* recap proposals submitted from voter */
+        <>Nothing to display yet. Come back later!</>
+      }
+
+      {/* proposal registration started */}
+      {step > 0 &&
+        /* recap proposals submitted from voter */
+        <>
+          <h3>My proposals</h3>
+          <>List or no proposal yet</>
+        </>
+      }
+      {step == 1 &&
+        /* input + button to add proposals */
+        <>
+          <Divider sx={{ my: 2 }} />
+          <h3>Add a proposal</h3>
+          <>form</>
+        </>
+      }
+
+      {/* vote started */}
+      {step > 2 && !hasVoted &&
+        <>
+          <Divider sx={{ my: 2 }} />
+          <h3>Vote</h3>
+          <FormControl fullWidth>
+            <InputLabel id="proposition-id" fullWidth>Vote For</InputLabel>
+            <Select
```

Comment:
le comportement du composant n'est pas ouf (déjà le cas dans mes premiers tests), quand on focus sur le select, le "Vote For" se calle sur le cadre, ça donne l'impression qu'il est rayé et ça fait assez moche je trouve. 
je n'attends pas un fix, ça doit être la lib, mais c'est étonnant / déceptif

Reply:
Oui j'ai vu ça. On se le garde sous le coude, pour plus tard mais effectivement il y a un risque que ce soit la librairie le problème.